### ISSUE TITLE
Expose "allow_huge" in chi2_shift function

### ISSUE THREAD

**Issue by adrtsc** [keflavich/image_registration#60]
Hi, I often use chi2_shift to register images and get good results with it. Sometimes, I try to register relatively large images and I get the following error message:

```
ValueError: Size Error: Arrays will be 3.2G.  Use allow_huge=True to override this exception.
```

Therefore, I often have to use downscaled versions of the images, which introduces small shifts in the results. Unfortunately, "allow_huge" is not exposed directly from the chi2_shift function. Chi2_shift seems to call on image_registration/fft_tools/correlate2d, which in turn calls convolution/convolve/convolve_fft from astropy. 

Would it be reasonable to expose "allow_huge" in image_registration/fft_tools/correlate2d and image_registration/chi2_shifts/chi2_shift?

**Comment by keflavich** [comment]
Yes, that's a good idea.  Would you be willing to submit a pull request doing just that?

**Comment by adrtsc** [comment]
Great! Sure, I'll submit a PR with the changes next week.
